src/lib/features/client-feature-toggles/client-feature-toggle.controller.ts

```diff
@@ -162,6 +165,32 @@ export default class FeatureController extends Controller {
     private async resolveFeaturesAndSegments(
         query?: IFeatureToggleQuery,
     ): Promise<[FeatureConfigurationClient[], IClientSegment[]]> {
+        if (this.flagResolver.isEnabled('deltaApi')) {
```

Comment:
Only if flag on, do this logic, otherwise go default. We will start very gradual rollout.